### PR TITLE
Remove left sidebar on myEYPD page

### LINKED ISSUE #482: myEYPD: Remove left navigation
This ticket has evolved. @kathreenriel has written the revision to this ticket 

**Description:**
With the activation of users profile now visible to administrators the links in the left navigation have also increased on for registered users myEYPD page. 
**The Problem:** 
With 13 additional links now visible to the user, the myEYPD appears to be more cluttered with text. 
**The User Story:**
As product owner I aim to keep the myEYPD page focused on the management of training events. The links on the left vertical navigation could potentially distract the user from using the tools in the main body of this page. 

**Suggested Steps to Resolve**
Remove the left nav bar just from the myEYPD page. 
screen shot 2018-12-03 at 10 32 53 am

**Measures of Success**
The myEYPD presents a "clean" interface that supports the Learner to focus on the management of their training events. 

**Version:**
WP 4.9.7

### PR DESCRIPTION
**Pull request by brettjforsyth**
Resolves #482. Tested the page isolation logic on all event pages and didn't find any issues. However this still relies on a page name (the page id reported was 0 which seems wrong or fragile) which also seems a bit fragile long term.

### CODE REVIEW

File: members/single/plugins.php [hunk 1]
```diff
@@ -15,7 +15,9 @@
 
 <?php get_header( 'buddypress' ); ?>
 
-<div id="content" role="main" class="<?php do_action( 'content_class' ); ?>">
+
+
+<div id="content" role="main" class="<?php ( ! is_page() && ! $pagename === 'events' ) ? do_action( 'content_class' ) : do_action( 'no_sidebar_content_classes' ); ?>">
```
Comment (bdolor) [inline members/single/plugins.php]: the logic here can be triggered on the professional interests page as well. Is that intentional?

if we're adding our own action, should 'namespace' it / prefix it with `eypd_`

have you also looked into the value of hooking into the existing `content_class` action?

File: functions.php [hunk 2]
```diff
@@ -1733,3 +1733,17 @@ function eypd_copyright_menu() {
...
+|--------------------------------------------------------------------------
+| Layout Overrides
+|--------------------------------------------------------------------------
+|
+|
+|
+|
+*/
+function eypd_no_sidebar_content_classes() {
+	print 'column sixteen';
```
[the remark this answers is not in the document]
Comment (brettjforsyth) [reply functions.php]: Yeah it felt weird

File: members/single/plugins.php (hunk 1 above)
Comment (brettjforsyth) [reply members/single/plugins.php]: Content_class wasn’t easily hookable as it just prints some text so I decided against it.

Comment (brettjforsyth) [reply members/single/plugins.php]: Shit thought I had added eypd

What’s the URL for pro interests. It was unintentional

Comment (bdolor) [reply members/single/plugins.php]: {domain}/members/{your-name}/professional-interests - can also get there via 'recommend events' button on myEYPD

Comment (brettjforsyth) [reply members/single/plugins.php]: Fixed this for pro-interests. I think it should also fix it for other instanaces as well if there are any. The test is more specific now.

**Pull request merged**